credentialdigger/client_sqlite.py

```diff
@@ -181,6 +192,60 @@ def add_discoveries(self, discoveries, repo_url):
                 state=d['state']
             ), discoveries)
 
+    def add_embedding(self, discovery_id, embedding=None, repo_url=''):
```

Comment:
I assume this method (and add_embeddings) can be moved (maybe partially, maybe wholly) in the parent class

Reply:
It's already largely in the parent class:
      return super().add_embedding(query,
                                     discovery_id,
                                     snippet,
                                     embedding_str,
                                     repo_url)